web/src/components/Request.tsx

```diff
@@ -244,6 +248,51 @@ export const RequestAccessInstructions: React.FC = () => {
               {props.children}
             </Text>
           ),
+          code: (props) => {
+            const { children, node } = props;
+            let value = "";
+            if (node.children.length == 1 && node.children[0].type == "text") {
+              value = node.children[0].value;
+            }
+
+            const { hasCopied, onCopy } = useClipboard(value);
+            return (
+              <Stack>
+                <Code
+                  padding={0}
+                  bg="white"
+                  borderRadius="8px"
+                  borderColor="neutrals.300"
+                  borderWidth="1px"
+                >
+                  <Flex
+                    borderColor="neutrals.300"
+                    borderBottomWidth="1px"
+                    py="8px"
+                    px="16px"
+                    minH="36px"
+                  >
+                    <Spacer />
+                    <IconButton
+                      variant="ghost"
+                      h="20px"
+                      icon={hasCopied ? <CheckIcon /> : <CopyIcon />}
+                      onClick={onCopy}
+                      aria-label={"Copy"}
+                    />
+                  </Flex>
+                  <Text
+                    overflowX="auto"
+                    color="neutrals.700"
+                    padding={4}
+                    whiteSpace="pre"
```

Comment:
I think this will fix the wrapping but I don't currently have a dev environment to test this in (need to reinstall Go on my laptop)

```suggestion
                    whiteSpace="pre-wrap"
```